<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PromotionEditDialog</class>
 <widget class="QDialog" name="PromotionEditDialog">
  <property name="minimumWidth">
   <number>400</number>
  </property>
  <property name="windowTitle">
   <string>Маркетинговая акция</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing"><number>8</number></property>
     <property name="horizontalSpacing"><number>12</number></property>
     <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
     <property name="labelAlignment"><set>Qt::AlignRight|Qt::AlignVCenter</set></property>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="placeholderText">
        <string>Например: Скидка 10%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Описание:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="descriptionEdit">
       <property name="placeholderText">
        <string>Необязательно</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="percentLabel">
       <property name="text">
        <string>Скидка, %:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="percentSpin">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sumLabel">
       <property name="text">
        <string>Скидка, ₽:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="sumSpin">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.990000000000000</double>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
